shivatmica/41_first_missing_positive.py

```diff
@@ -1,52 +0,0 @@
-class Problem41:
-    """
-    Class for First Missing Positive (#41)
-    # https://leetcode.com/problems/first-missing-positive/
-    """
-    def firstMissingPositive(self, nums: List[int]) -> int:
-        """
-        Returns the smallest positive number that isn't included in the inputted list
-        Time complexity: O(n)
-        Space complexity: O(1)
-        
-        Parameters
-        ----------
-        nums : List[int]
-            The inputted list, an unsorted integer array
-            
-        Returns
-        -------
-        i : int
-            The index of the first missing positive number
-        """
-        
-        # adding 0 to the end of the list in the case that all the numbers are consecutive and smallest positive numbers
-        # the function will then return the value that is greater than all of these values
-        nums.append(0) 
```

Comment:
Please review [this PR](https://github.com/brucechanglongxu/practice-problems/pull/15) for feedback, as you're making some of the same mistakes. In particular, we shouldn't be modifying `nums` (why not?)